forms/daily.json

```
@@ -45,10 +45,4 @@
     "type": "emojirange",
     "validation": "required"
   },
```

Le contenu d'un objet JSON ne doit jamais se terminer par une virgule.

```suggestion
  }
```